src/core/tests/opset.cpp

```diff
@@ -69,7 +70,8 @@ INSTANTIATE_TEST_SUITE_P(opset,
                                          OpsetTestParams{ov::get_opset9, 173},
                                          OpsetTestParams{ov::get_opset10, 177},
                                          OpsetTestParams{ov::get_opset11, 177},
-                                         OpsetTestParams{ov::get_opset12, 178}),
+                                         OpsetTestParams{ov::get_opset12, 178},
+                                         OpsetTestParams{ov::get_opset13, 178}),
```

Comment:
```suggestion
                                         OpsetTestParams{ov::get_opset13, 179}),
```
?

Reply:
This is a number of the operations in the opset, as the initial point it's the same value as the Opset12.
But yes it will be updated +1 with introduction of each new operator.